concent_api/core/tests/test_auth_report_computed_task.py

```diff
@@ -0,0 +1,892 @@
+from base64                 import b64encode
+
+from django.test            import override_settings
+from django.test            import TestCase
+from django.urls            import reverse
+from freezegun              import freeze_time
+from golem_messages         import dump
+from golem_messages         import load
+from golem_messages         import message
+import dateutil.parser
+
+from core.models            import MessageAuth
+from utils.testing_helpers  import generate_ecc_key_pair
+
+
+(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)                         = generate_ecc_key_pair()
+(PROVIDER_PRIVATE_KEY, PROVIDER_PUBLIC_KEY)                       = generate_ecc_key_pair()
+(REQUESTOR_PRIVATE_KEY, REQUESTOR_PUBLIC_KEY)                     = generate_ecc_key_pair()
+(DIFFERENT_PROVIDER_PRIVATE_KEY, DIFFERENT_PROVIDER_PUBLIC_KEY)   = generate_ecc_key_pair()
+(DIFFERENT_REQUESTOR_PRIVATE_KEY, DIFFERENT_REQUESTOR_PUBLIC_KEY) = generate_ecc_key_pair()
```

Comment:
This is not something to fix now but a remark to keep in mind in the future:

`CONCENT_PRIVATE_KEY` and `CONCENT_PUBLIC_KEY` were originally defined as global constants at the top of the file only because we needed to use them in `override_settings` on multiple test classes. The other keys should not be created here. I think I added them at the top in my original integration tests and that's why they continue to be added here but we really should start defing them inside `setUp()` in new tests (they should be named in lower case of course when they're moved).

Reply:
:+1: I generally looked for a way to move generating CONCENT_PRIVATE_KEY and CONCENT_PUBLIC_KEY to `setUpClass` to override settings there but it didn't worked out as I have expected. Please note that in `test_auth_get_task_result.py` this  provider and requestor keys are generated in setup, so I will move `DIFFERENT_` keys in that file there too.

Reply:
Yeah, I saw that. But I'm only taking about keys other than `CONCENT_...` ones. I think that `CONCENT_...` keys should stay where the are. It's only the `PROVIDER_...` keys, `REQUESTOR_...` keys and others that should be moved.